metis_client/dtos/__init__.py

```diff
@@ -14,11 +14,12 @@
     MetisDataSourceContentOnlyDTO,
     MetisDataSourceDTO,
 )
-from .error import MetisErrorDTO
+from .error import MetisErrorDTO, MetisErrorErrorDTO
```

Comment:
`ErrorError` 😭

Reply:
No, I’m really against having `MetisErrorDTO` and `MetisErrorErrorDTO`

Reply:
Too many levels of data nesting 🤣